src/js/outreach/views/dialog.scss

```diff
@@ -10,104 +10,38 @@
   text-transform: uppercase;
 }
 
-.icon {
-  font-size: 32px;
-}
-
 .dialog {
   background: $white;
   border: 1px solid $black-90;
   border-radius: 4px;
   color: $black-20;
-  font-size: 24px;
+  font-size: 20px;
   line-height: 1.4;
   padding: 24px;
-  text-align: left;
 }
 
 .dialog__icon {
   color: $black-20;
-  margin-bottom: 24px;
-  text-align: left;
-
-  .icon {
-    font-size: 48px;
-    width: 1em;
-  }
-}
-
-.dialog__icon--success {
-  color: $success-color;
-  margin: 0 auto 24px;
-}
-
-.dialog__icon--warn {
-  color: $warning-color;
-  margin: 0 auto 24px;
-}
-
-.dialog__icon--error {
-  color: $error-color;
   margin: 0 auto 24px;
-}
-
-.dialog__input {
-  border: 1px solid $black-90;
-  border-radius: 4px;
-  color: $black-20;
-  margin: 32px 0;
-  padding: 0 16px;
-  position: relative;
-  text-align: center;
-  width: 100%;
 
-  &.has-error {
-    border-color: $error-color;
-    color: $error-color;
-    margin-bottom: 6px;
+  &.dialog__icon--success {
```

Comment:
I don't think the `&`s are necessary here.  these classes look to be BEM'd enough that they do not need to be nested.  They just need to be either after the `dialog_icon` in the cascade or since it's all in the same file these could extend the base such that only this class is necessary in the DOM.  Ideally `&` is saved for adding states generally prefixed by something like `&.has-`, `&.is-` etc

Reply:
I went the `@extend` route. Works well. I like that we now only need the `.dialog__icon--` classname in the dom. Instead of needing `.dialog__icon .dialog__icon--`. Good call 👍